v2/public/localscripts/calculationQuestion/SimpleClient.js

```diff
@@ -297,7 +297,7 @@ define(["jquery", "underscore", "backbone", "rivets", "PrairieTemplate"], functi
         this.options = _.defaults(options || {}, {
             questionTemplate: "",
             answerTemplate: "",
-            submissionTemplate: "",
+            submissionTemplate: "Submission not shown.",
```

Comment:
Also change this to `Saved answer`? Are there other places we that the student will see the word "submission"?